submissions/andysmokk/classic-frogger-game/app.js

```diff
@@ -0,0 +1,108 @@
+// Enemies our player must avoid
+const POSITION_PLAYER_X = 203;
+const POSITION_PLAYER_Y = 390;
+const POSITION_ENEMIES_X = -200;
+const POSITION_ENEMY_ONE_Y = 62;
+const POSITION_ENEMY_TWO_Y = 145;
+const POSITION_ENEMY_THREE_Y = 228;
+const BLOCK_HEIGHT = 83;
+const BLOCK_WIDTH = 101;
+const MIN_SPEED = 100;
+const MAX_SPEED = 400;
+
+var Enemy = function (x, y) {
+  this.x = x;
+  this.y = y;
+  this.speed = this.randomInteger();
+  this.sprite = "images/enemy-bug.png";
+};
+
+Enemy.prototype.update = function (dt, gamer) {
+  this.x += this.speed * dt;
+
+  if (this.x > 500) {
+    this.x = POSITION_ENEMIES_X;
+    this.speed = this.randomInteger();
+  }
+
+  if (
+    gamer.y < this.y + 50 &&
+    gamer.y + 50 > this.y &&
+    gamer.x < this.x + 75 &&
+    gamer.x + 75 > this.x
+  ) {
+    gamer.initialPositionPlayer();
+  }
+};
```

Comment:
This shouldn't work unless you changed the `engine`.
Please re-do as changing the `engine` is a no-go here.
Imagine, you are on a commercial project and opt to change shared code when the problem is solvable without doing this.
The requirements contain a hint on how to resolve the issue properly. Just need to read carefully.

Reply:
Fixed.